platform/security/sts/security-sts-attributequeryclaimshandler-common/src/main/java/org/codice/ddf/security/claims/attributequery/common/AttributeQueryClaimsHandler.java

```diff
@@ -226,7 +243,71 @@ protected ProcessedClaim createSingleValuedClaim(String claimType, String claimV
      */
     protected AttributeQueryClient createAttributeQueryClient(SimpleSign simpleSign,
             String externalAttributeStoreUrl, String issuer, String destination) {
-        return new AttributeQueryClient(simpleSign, externalAttributeStoreUrl, issuer, destination);
+        return new AttributeQueryClient(createDispatcher(createService()),
+                simpleSign,
+                externalAttributeStoreUrl,
+                issuer,
+                destination);
+    }
+
+    /**
+     * Creates a dynamic service from the provided wsdl location.
+     */
+    protected Service createService() {
+        Service service = null;
+        if (StringUtils.isNotBlank(wsdlLocation) && StringUtils.isNotBlank(serviceName)) {
+            service = Service.create(this.getClass()
+                    .getClassLoader()
+                    .getResource(wsdlLocation), QName.valueOf(serviceName));
```

Comment:
We should not be relying on a WSDL that is compiled with this code. The WSDL will be remote in almost every situation.